src/main/java/com/google/alpollo/SentimentServlet.java

```diff
@@ -17,11 +19,19 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String lyrics = request.getParameter("lyrics");
     AnalysisHelper helper = new AnalysisHelper();
-    Sentiment sentiment = helper.getSentiment(lyrics);
-    SongSentiment songSentiment = new SongSentiment(sentiment.getScore(), sentiment.getMagnitude());
-    
-    String json = gson.toJson(songSentiment);
-    response.setContentType("application/json;");
-    response.getWriter().println(json);
+    try {
+      Sentiment sentiment = helper.getSentiment(lyrics);
+      SongSentiment songSentiment = new SongSentiment(sentiment.getScore(), sentiment.getMagnitude());
+
+      String json = gson.toJson(songSentiment);
+      response.setContentType("application/json;");
+      response.getWriter().println(json);
+    } catch (JsonIOException | JsonSyntaxException jsonException){
+      response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR,
+              "Json file with configuration info is invalid");
+    } catch (Exception e) {
+      response.sendError(HttpServletResponse.SC_NOT_FOUND,
+              "Internal error occurred or config.json file not found");
+    }
```

Comment:
This `SentimentServlet` doesn't need to know details about the errors that can happen in the `ConfigHelper`.  `getSentiment` should only throw one type of exception (maybe `IllegalStateException`), and then you can catch it here and return a single `response.sendError`, which can reuse the exception message.